float32 roll
float32 pitch
float32 yaw
uint16 motorCmd1
uint16 motorCmd2
uint16 motorCmd3
uint16 motorCmd4
uint8 onboardControllerType

